MANIFEST.in

```diff
@@ -1,3 +1,8 @@
 recursive-include tljh_repo2docker/templates/ *
 recursive-include tljh_repo2docker/static/ *
 recursive-include tljh_repo2docker/tests/ *
+recursive-include frontend/ *
```

Comment:
Maybe it could also be useful to switch to `hatch` and drop this `MANIFEST.in`? (could be done / tracked separately)

Reply:
https://github.com/plasmabio/tljh-repo2docker/issues/71